run_and_analyze.sh

```diff
@@ -96,6 +96,21 @@ case "$scheduler" in
         collective_string=$(echo $collectives | sed 's/ /,/g')
         qsub -hold_jid $collective_string $work_dir/analyze.sh -c $config_file
     ;;
+    "PBS")
+        python coltune_script.py $config_file pbs
+        if [ $? -ne 0 ]; then
+            echo "Failed to create job scripts. Exiting.."
+            exit 1;
+        fi
+
+        for collective in ${collectives// / } ; do
+            pushd $work_dir/output/$collective/
+            qsub ${collective}_coltune.sh
+            popd
+        done
+        #jobIDs=$(echo "after:$collectives" | sed 's/ /:/g')
```

Comment:
Still gotta remove this commented code

Reply:
Thanks for pointing out. Removed this left over from an attempt to create a job dependency chain with PBS ...